src/dispatch/feedback/service/views.py

```diff
@@ -12,3 +19,31 @@
 def get_feedback_entries(commons: CommonParameters):
     """Get all feedback entries, or only those matching a given search term."""
     return search_filter_sort_paginate(model="ServiceFeedback", **commons)
+
+
+@router.get("/{service_feedback_id}", response_model=ServiceFeedbackRead)
+def get_feedback(db_session: DbSession, service_feedback_id: PrimaryKey):
+    """Get a feedback entry by its id."""
+    feedback = get(db_session=db_session, service_feedback_id=service_feedback_id)
+    if not feedback:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail=[{"msg": "A feedback entry with this id does not exist."}],
+        )
+    return feedback
+
+
+@router.delete(
+    "/{service_feedback_id}",
+    response_model=None,
+    dependencies=[Depends(PermissionsDependency([OrganizationAdminPermission]))],
```

Comment:
Should we allow the creator of the feedback to delete it as well, unless it's anonymous?